### PR TITLE
feat: Email verify OTP for owner and user registration

### PR DESCRIPTION
**Pull request by Haider8**

### CODE REVIEW

File: app/core/config.py [hunk 1]
```diff
@@ -18,3 +19,19 @@
...
+MAIL_PORT = int(os.getenv("MAIL_PORT")) or 465
+MAIL_USERNAME = os.getenv("MAIL_USERNAME")
+MAIL_PASSWORD = os.getenv("MAIL_PASSWORD")
+MAIL_SENDER = os.getenv("MAIL_SENDER")
+if tls := os.getenv("MAIL_USE_TLS"):
+    if tls == "True":
+        MAIL_USE_TLS = True
+    elif tls == "False":
+        MAIL_USE_TLS = False
+        warning("Setting MAIL_USE_TLS to `False`")
```
Comment (iamareebjamal) [inline app/core/config.py:35]: Python dotenv has a feature to cast variables into booleans, but that might be done in another PR

**Review by iamareebjamal — APPROVED** [review]

File: app/core/config.py [hunk 2]
```diff
@@ -18,3 +19,19 @@
...
 
 ACCESS_TOKEN_EXPIRE_MINUTES = 60 * 24 * 8  # 60 minutes * 24 hours * 8 days = 8 days
+
+# Configs for sending emails
+MAIL_SERVER = os.getenv("MAIL_SERVER")
+MAIL_PORT = int(os.getenv("MAIL_PORT")) or 465
+MAIL_USERNAME = os.getenv("MAIL_USERNAME")
+MAIL_PASSWORD = os.getenv("MAIL_PASSWORD")
+MAIL_SENDER = os.getenv("MAIL_SENDER")
+if tls := os.getenv("MAIL_USE_TLS"):
```
[the remark this answers is not in the document]
Comment (Haider8) [reply app/core/config.py:30]: @iamareebjamal Looks like they don't support python 3.8
